evap/staff/tests/test_importers.py

```diff
@@ -122,16 +122,25 @@ def test_import_makes_inactive_user_active(self):
 class TestEnrollmentImporter(TestCase):
     filename_valid = os.path.join(settings.BASE_DIR, "staff/fixtures/test_enrollment_data.xls")
     filename_valid_degree_merge = os.path.join(settings.BASE_DIR, "staff/fixtures/test_enrollment_data_degree_merge.xls")
+    filename_valid_import_names = os.path.join(settings.BASE_DIR, "staff/fixtures/test_enrollment_data_import_names.xls")
     filename_invalid = os.path.join(settings.BASE_DIR, "staff/fixtures/invalid_enrollment_data.xls")
     filename_random = os.path.join(settings.BASE_DIR, "staff/fixtures/random.random")
 
+    maxDiff = None
+
```

Comment:
?

Reply:
This was useful as the diff generated by `assertCountEqual` got sometimes too long to display with the default settings. `None` always displays the full diff; if you rather want to have a fixed `maxDiff`, I can look which value is a good fit here

Reply:
ahh, i just didn't know that, looked like an unused variable for me.
Maybe add a comment? Idk.

Reply:
We decided against a comment as it is a property of the standard library [`TestCase.maxDiff`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.maxDiff).

Reply:
most evap developers probably don't know the library by heart and i also found it unexpected when first seeing it. adding a comment shouldn't hurt so i'd be happy if you could do that.

Reply:
Now removed as the errors are categorized and the diff does not grow big enough anymore